packages/jellyfish-transaction-builder/src/txn/txn_builder_icxorderbook.ts

```diff
@@ -0,0 +1,18 @@
+import { OP_CODES, Script, TransactionSegWit, ICXCreateOrder } from '@defichain/jellyfish-transaction'
+import { P2WPKHTxnBuilder } from './txn_builder'
+
+export class TxnBuilderICX extends P2WPKHTxnBuilder {
```

Comment:
equivalent names as jellyfish-core
```suggestion
export class TxnBuilderICXOrderBook extends P2WPKHTxnBuilder {
```